nodejs-bundle-webpack/nodejs-bundle-webpack.spec

```diff
@@ -0,0 +1,191 @@
+%global npm_name webpack
+
+Name: nodejs-%{npm_name}
+Version: 1.13.1
+Release: 1%{?dist}
+Summary: Packs CommonJs/AMD modules for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand. Support loaders to preprocess files, i.e. json, jade, coffee, css, less, ... and your custom stuff.
+License: MIT
+Group: Development/Libraries
+URL: https://github.com/webpack/webpack.git
+Source0: http://registry.npmjs.org/webpack/-/webpack-1.13.1.tgz
```

Comment:
The way I was doing it https://github.com/dLobatog/njs2rpm/commit/084b5187c3cfd2b54e9db17144c82c68007ec8a0 seems easier but it won't match the source. I think I can use the same list of packages I put in Provides and convert them to URLs here.